graphite_api/functions.py

```diff
@@ -2711,14 +2711,18 @@ def smartSummarize(requestContext, seriesList, intervalString, func='sum'):
 
     # Adjust the start time to fit an entire day for intervals >= 1 day
     requestContext = requestContext.copy()
+    # ISSUE #51: Added tzinfo to requestContext in graphite_api/app.py:328
+    tzinfo = requestContext.get('tzinfo', None)
```

Comment:
Bruno, I originally did the direct key request as you suggested, but the unit test failed for smartSummarize as it provided data directly to the function without any tzinfo key.  I figured I'd rather error on the side of caution and not change your unit tests as well ;-)

I can easily remove the comment, I just added it for notation...

Thanks!

--Geoff Thornton